actions/keeper_helpers.go

```diff
@@ -105,11 +105,11 @@ func DeployPerformanceKeeperContracts(
 	contractDeployer contracts.ContractDeployer,
 	networks *blockchain.Networks,
 	registrySettings *contracts.KeeperRegistrySettings,
-	blockRange, // How many blocks to run the test for
-	blockInterval, // Interval of blocks that upkeeps are expected to be performed
+	blockRange,     // How many blocks to run the test for
```

Comment:
maybe you forgot to configure Goland to run some linting on save?

Reply:
Yes, that was the case, it's fixed now :)